main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java

```diff
@@ -343,9 +344,12 @@ public boolean onOptionsItemSelected(final MenuItem item) {
                 return true;
             case R.id.menu_map_live:
                 mapOptions.isLiveEnabled = !mapOptions.isLiveEnabled;
+                if (mapOptions.isLiveEnabled) mapOptions.isStoredEnabled = true;
```

Comment:
Do we have a checkstyle rule that braces are needed? I'm not sure at the moment.